apple/OmnivoreKit/Sources/App/Views/WebReader/WebReaderContent.swift

```diff
@@ -45,10 +42,10 @@ struct WebReaderContent {
         </div>
         <script type="text/javascript">
           window.omnivoreEnv = {
-            "NEXT_PUBLIC_APP_ENV": "\(appEnv.rawValue)",
-            "NEXT_PUBLIC_\(appEnv.rawValue.uppercased())_BASE_URL": "\(appEnv.webAppBaseURL.absoluteString)",
-            "NEXT_PUBLIC_\(appEnv.rawValue.uppercased())_SERVER_BASE_URL": "\(appEnv.serverBaseURL.absoluteString)",
-            "NEXT_PUBLIC_\(appEnv.rawValue.uppercased())_HIGHLIGHTS_BASE_URL": "\(appEnv.highlightsServerBaseURL.absoluteString)"
+            "NEXT_PUBLIC_APP_ENV": "prod",
```

Comment:
Not used but if this isn't set to a valid value then the web assertions break.

Reply:
Got it, maybe we can slim things down a bit more in a future bundle. Not a big deal right now though.